includes/ucf-academic-calendar-feed.php

```diff
@@ -20,6 +20,9 @@ public static function get_feed( $args ) {
 
 			$feed_url       = $args['calendar_feed'];
 			$use_cache      = UCF_Acad_Cal_Config::get_option_or_default( 'cache_items' );
+			$offset         = isset( $args['offset'] ) ? intval( $args['offset'] ) : $args['default_offset'];
+			$count          = isset( $args['count'] ) ? intval( $args['count'] ) : $args['default_count'];
+			$isImportant    = $args['is_important'];
```

Comment:
Underscores are preferred over camelCase for PHP variable names (`$is_important`)